disconnect-when-backgrounded.js

```diff
@@ -13,10 +13,12 @@ if (Meteor.isCordova) {
     document.addEventListener('pause', function () { createDisconnectTimeout(); });
 }
 
+const currentPageIsNotExempt = () => { return disconnectVoids.indexOf(window.location.pathname) == -1; };
+
 function disconnectIfHidden() {
     removeDisconnectTimeout();
 
-    if (document.hidden) {
+    if (document.hidden && currentPageIsNotExempt()) {
```

Comment:
Can this be combined with the similar `disconnectVoids.indexOf(Router.current().route.getName()) < 0` line below?

Perhaps:
```
function currentPageIsNotExempt() {
  const pathName = Package["iron:router"] ? Router.current().route.getName() : window.location.pathname;
  return disconnectVoids.includes(pathName);
};
```

Reply:
Yeah sure, I'll take a look at it now.